### PR TITLE
Add OpenAI conversion

### PR DESCRIPTION
**Pull request by Jflick58**
Added OpenAI conversion functionality to the mdptoolbox examples. 

It's fully documented in the doctrings, and I tested it successfully. 

Unfortunately, it only supports discrete OpenAI gyms right now.

### CODE REVIEW

File: hiive/mdptoolbox/example.py [hunk 1]
```diff
@@ -402,3 +404,62 @@ def small():
...
+    You'll have to look at the source code of the environments for available kwargs; as it is not well documented.  
+
+    This function is used to generate a transition probability
+    (``A`` × ``S`` × ``S``) array ``P`` and a reward (``S`` × ``A``) matrix
+    ``R``.
+
+    Parameters
+    ---------
+    env_name : str
+        The name of the Open AI gy environment to model.
```
Comment (TEChopra1000) [inline hiive/mdptoolbox/example.py]: small typo - gy --> gym

Comment (Jflick58) [reply hiive/mdptoolbox/example.py]: Fixed in 952d47ab5aebeef39051df22c677d494c990c031

**Review by hiive — APPROVED** [review]

**Comment by ricka** [comment]
Would you mind cutting a new release so this is included?

**Comment by hiive** [comment]
On PyPi? Sure - I'll try and find time to do it this week.

Andrew

**Comment by mike-altonji** [comment]
These are awesome updates! +1 to ricka, could you update PyPi?

**Comment by hemanik** [comment]
@hiive, could you release a new version to PyPi?

**Pull request merged**